android-reset-keystore.md

```diff
@@ -0,0 +1,19 @@
+# Reseting your Android Upload key
+## Background
+All newer apps (and many others) that are live on the Play Store are opted into [Google Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?hl=en). With Play App Signing, Google maintains the signing key used to ultimately verify that your app is authentic. Meanwhile, when your app is built, it is signed with a separate key known as the "Upload key." Google Play checks your upload key when you go to publish your app to ensure the app binary is from you. If you lose this key, as long as you still have access to your Play Store account, you can reset it and continue publishing your app.
```

Comment:
```suggestion
[Google Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?hl=en) is a security service where Google manages the app signing key on behalf of you as a developer.  All newer apps (and many others) that are live on the Play Store are opted into Play App Signing. 

With Play App Signing, Google maintains the signing key used to verify that your app is authentic. Meanwhile, when your app is built, it is signed with a separate "Upload key". Google Play checks your upload key when you are going to publish your app to ensure the app binary is from you. If you lose this key, as long as you still have access to your Play Store account, you can reset it and continue publishing your app.
```

Let's divide this so that it doesn't read like a wall of text and is skimmable. I also added one line at the start to briefly explain what Play Signing is so that when someone is ready for the doc, they don't have to look here and there when they come across "Play App Signing" terminology.